### ISSUE TITLE
Zufällige Sortierung auf mehreren Seiten

### ISSUE THREAD

**Issue by larsUE** [puzzle/quartieridee#65]
Ich schreibe das mal hier hin, obwohl es wohl eher zu meta-decidim passt. Wenn man viele Ideen hat und die sich auf mehrere Seiten verteilen ist die zufällige Sortierung enorm umständlich. Wenn ich eine Seite weiterblättere, so werden alle Ideen neu sortiert, so dass ich im Endeffekt immer ähnliche Ideen sehe, während ich andere gar nicht sehe. Kann dies irgendwie verhindert werden?

**Comment by bsantschi** [comment]
Vorschläge von Lars: 
- Per default gleich alle 100 Ideen anzeigen, da die Ideenphase nun beendet ist und nicht mehr kommen.
- oder der Filter „Kürzlich hinzugefügt“ als Standardeinstellung.

**Comment by carlobeltrame** [comment]
Zweiteres ist eine sehr kleine Sache und ist implementiert in https://github.com/puzzle/quartieridee/commit/7efc5b41e53116e82215a32cae43d7ae5b70459c
Momentan ist aber noch etwas anderes am Code kaputt, das müssten wir erst noch fixen wenn wir das ausrollen wollen.

**Comment by larsUE** [comment]
Hab mal ein Bug-Report erfasst: https://meta.decidim.org/processes/bug-report/f/210/proposals/16196?order=recent

Oder @carlobeltrame hast du das Gefühl, dieses Verhalten tritt nur auf der Quartieridee auf?

**Comment by carlobeltrame** [comment]
Nein, das ist das Standardverhalten von decidim, und in meinen Augen ein Bug. Finde ich gut dass du das eröffnet hast :+1:
